File: app/src/main/java/life/plank/juna/zone/data/network/model/Thumbnail.java
package life.plank.juna.zone.data.network.model;

import lombok.Data;

/**
 * Created by plank-arfaa on 06/02/18.
 */

@Data
public class Thumbnail {
    private String imageUrl;
    private int imageHeight;
    private int imageWidth;

    public Thumbnail() {
    }

    public Thumbnail(String imageUrl, int imageHeight, int imageWidth) {
        this.imageUrl = imageUrl;
        this.imageHeight = imageHeight;
        this.imageWidth = imageWidth;
    }

}

File: app/src/main/java/life/plank/juna/zone/data/model/Thumbnail.kt
package life.plank.juna.zone.data.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Thumbnail(
        var imageUrl: String = "",
        var imageHeight: Int = 0,
        var imageWidth: Int = 0
) : Parcelable